src/oralb_ble/parser.py

```diff
@@ -341,19 +342,24 @@ async def async_poll(self, ble_device: BLEDevice) -> SensorUpdate:
             battery_payload = await client.read_gatt_char(battery_char)
             pressure_char = client.services.get_characteristic(CHARACTERISTIC_PRESSURE)
             pressure_payload = await client.read_gatt_char(pressure_char)
+        except BleakError as err:
+            _LOGGER.debug(f"Reading gatt characters failed with err: {err}")
```

Comment:
Should we log at warning level so they know why it's missing?

Reply:
Sure - I can make that change. My only hesitation is I think sometimes these gatt char functions just fail. At least they did while I was testing.

That being said, potentially it was just while I was testing, we can make the change and see.

If we end up getting flooded by issues of people getting these warnings maybe we bump it back down to debug? Not sure

Reply:
https://github.com/Bluetooth-Devices/bleak-retry-connector/blob/5ab0eaf91f28320052ce72315ec50276ef2ba407/tests/test_init.py#L1433

Reply:
It looks good. I'll try to find some time to test with it later today

Reply:
Sounds good - thanks